README.md

```diff
@@ -561,6 +562,19 @@ val disabled = audioVideo.realtimeSetVoiceFocusEnabled(false) // disabling Amazo
 Custom video source allows you to control the video, such as applying a video filter. For more details, see [Custom Video](https://github.com/aws/amazon-chime-sdk-ios/blob/master/guides/custom_video.md).
 
 
+## Frequently Asked Questions
+
+Refer to [General FAQ](https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html) for Amazon Chime SDK.
+
+### Debugging
+
+#### How can I get Amazon Chime SDK logs for debugging?
+Applications can get logs from Chime SDK by passing instances of Logger when creating [MeetingSession](https://github.com/aws/amazon-chime-sdk-ios/blob/master/AmazonChimeSDK/AmazonChimeSDK/session/MeetingSession.swift#L13). Amazon Chime SDK has some default implementations of logger that your application can use, such as [ConsoleLogger](https://github.com/aws/amazon-chime-sdk-ios/blob/master/AmazonChimeSDK/AmazonChimeSDK/utils/logger/ConsoleLogger.swift) which logs into console. `ConsoleLogger` is set to `INFO` level as default. Therefore, in order to get all logs, including media logs, create logger by following:
+```swift
+logger = ConsoleLogger(name: "logger", level: .DEFAULT)
```

Comment:
`.DEFAULT` is info, we need to state `verbose level`, as in Android's FAQ, right?

Reply:
We use `.DEFAULT` as VERBOSE in iOS. Refer [LogLevel](https://github.com/aws/amazon-chime-sdk-ios/blob/master/AmazonChimeSDK/AmazonChimeSDK/utils/logger/LogLevel.swift).